web/src/services/i18n/translations/en.yml

```
@@ -48,6 +48,29 @@ A To Z: A To Z
 Z To A: Z To A
 Size Small to Large: Size Small to Large
 Size Large to Small: Size Large to Small
+Upload File: Upload File
+Assets: Assets
+'Sort:': Sort:'
```

```suggestion
'Sort:': 'Sort:'
```
Do we really need `:` included?